examples/server/public/index.html

```diff
@@ -195,6 +297,27 @@
       }
     }
   </style>
+  <script>
+    window.ZeroMdConfig = {
+      markedUrl: 'https://cdn.jsdelivr.net/gh/markedjs/marked@4/marked.min.js',
+      prismUrl: [
+        ['https://cdn.jsdelivr.net/npm/prismjs@1.29.0/prism.min.js', 'data-manual'],
+        'https://cdn.jsdelivr.net/npm/prismjs@1.29.0/plugins/autoloader/prism-autoloader.min.js',
+      ],
+      cssUrls: window.matchMedia("(prefers-color-scheme: light)").matches
+        ? [
+          'https://cdn.jsdelivr.net/npm/github-markdown-css@5.5.0/github-markdown.min.css',
+          'https://cdn.jsdelivr.net/gh/katorlys/prism-theme-github/themes/prism-theme-github-light.css'
+        ]
+        : [
+          'https://cdn.jsdelivr.net/npm/github-markdown-css@5.5.0/github-markdown.min.css',
+          'https://cdn.jsdelivr.net/gh/katorlys/prism-theme-github/themes/prism-theme-github-dark.css'
+        ],
+      // Host styles that apply to each instance
+      hostCss: ':host{display:block;position:relative;contain:content;}:host([hidden]){display:none;}'
+    }
+  </script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/zero-md@2.5.3/dist/zero-md.min.js"></script>
```

Comment:
Doing this will make it hard to use the server entirely offline.

Reply:
makes sense, I have updated reps.sh to cache the dependencies locally for all files.